tests/ProjectTemplatesTest.php

```diff
@@ -0,0 +1,117 @@
+            if ($member['id'] === $this->normalUser['id']) {
+                $this->client->removeUserFromMaintainer($this->testMaintainerId, $member['id']);
+            }
+            if ($member['id'] === $this->superAdmin['id']) {
+                $this->client->removeUserFromMaintainer($this->testMaintainerId, $member['id']);
+            }
+        }
+        $organizations = $this->client->listMaintainerOrganizations($this->testMaintainerId);
+        foreach ($organizations as $organization) {
+            $this->client->deleteOrganization($organization['id']);
+        }
+        $this->organization = $this->client->createOrganization($this->testMaintainerId, [
+            'name' => 'My org',
+        ]);
+        $this->client->addUserToOrganization($this->organization['id'], ['email' => 'spam+spam@keboola.com']);
+        $this->client->removeUserFromOrganization($this->organization['id'], $this->superAdmin['id']);
+    }
+
+    public function testSuperAdminCanViewAndListProjectTemplates()
+    {
+        $templates = $this->client->getProjectTemplates();
+
+        $this->assertGreaterThan(0, count($templates));
+
+        $filteredTemplates = array_filter($templates, function ($item) {
+            if ($item['id'] !== self::TEST_PROJECT_TEMPLATE_STRING_ID) {
+                return false;
+            }
+            return true;
+        });
+        $this->assertGreaterThan(0, count($filteredTemplates));
+
+        $templateDetail = $this->client->getProjectTemplate(self::TEST_PROJECT_TEMPLATE_STRING_ID);
+
+        $this->assertEquals($templateDetail, current($filteredTemplates));
+
+    }
+
+    public function testOrganizationAdminCanViewAndListProjectTemplates()
+    {
+        $this->client->addUserToOrganization($this->organization['id'], ['email' => $this->normalUser['email']]);
+
+        $templates = $this->normalUserClient->getProjectTemplates();
+
+        $filteredTemplates = array_filter($templates, function ($item) {
+            if ($item['id'] !== self::TEST_PROJECT_TEMPLATE_STRING_ID) {
+                return false;
+            }
+            return true;
+        });
+        $this->assertGreaterThan(0, count($filteredTemplates));
+
+        $templateDetail = $this->client->getProjectTemplate(self::TEST_PROJECT_TEMPLATE_STRING_ID);
+
+        $this->assertEquals($templateDetail, current($filteredTemplates));
+    }
+
+    public function testSuperAdminCanViewHiddenProjectTemplate()
+    {
+        $template = $this->client->getProjectTemplate(self::TEST_HIDDEN_PROJECT_TEMPLATE_STRING_ID);
+
+        $this->assertArrayHasKey('id', $template);
+        $this->assertArrayHasKey('name', $template);
+        $this->assertArrayHasKey('description', $template);
+        $this->assertArrayHasKey('expirationDays', $template);
+        $this->assertArrayHasKey('hasTryModeOn', $template);
+    }
+
+    public function testOrganizationAdminCannotViewHiddenProjectTemplate()
```

Comment:
Tenhle test je spatne. Tady vubec nepridavas toho `normalUsera` do organizace. Mel bys ho na zacaktu pridat stejne jako u `testOrganizationAdminCanViewAndListProjectTemplates`.

Reply:
jj to by mělo - je to tam